src/components/implementation/no_interface/vkernel/vk_server.c

```diff
@@ -0,0 +1,47 @@
+
+#include "vk_types.h"
```

Comment:
These are headers in the vkernel directory and only applicable to current dir, so I think `"<local_header>"` is the way to do it.